patients/service.go

```diff
@@ -69,6 +70,13 @@ func (s *service) Update(ctx context.Context, update PatientUpdate) (*Patient, e
 		}
 	}
 
+	if shouldSetLastRequestedDexcomConnect(*existing, update) {
```

Comment:
Should this be handled in a separate function/endpoint instead of overloading the update? This way it will be similar to how upload reminders are handled.